src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -437,6 +439,40 @@ public static File createTempFileInDirectory(final String name, String extension
         }
     }
 
+    /**
+     * Creates a temp path that will be deleted on exit.
+     *
+     * This will also mark the corresponding Tribble/Tabix/BAM indices matching the temp file for deletion.
+     *
+     * @param name Prefix of the file.
+     * @param extension Extension to concat to the end of the file.
+     *
+     * @return A file in the temporary directory starting with name, ending with extension, which will be deleted after the program exits.
+     */
+    public static Path createTempPath(String name, String extension) {
```

Comment:
Would be good to have a standalone unit test for this method in `IOUtilsUnitTest` (since the test in `CommandLineProgramIntegrationTest` is testing the `--tmp-dir` argument rather than this method directly.

Reply:
Done.